manuscript/ha-docker-swarm/shared-storage-ceph.md

```diff
@@ -51,7 +51,7 @@ docker run -d --net=host \
 -e MON_IP=192.168.31.11 \
 -e CEPH_PUBLIC_NETWORK=192.168.31.0/24 \
 --name="ceph-mon" \
-ceph/daemon mon
+ceph/daemon:latest-mimic mon
```

Comment:
Per this comment in Discourse (https://discourse.geek-kitchen.funkypenguin.co.nz/t/shared-storage-ceph-funky-penguins-geek-cookbook/47/81), updated this one to use the `latest-mimic` tag. Prior to doing that, the MON container would not start.